examples/react-router/src/pages/Login.tsx

```diff
@@ -1,14 +1,16 @@
-import * as React from 'react'
-import { useCallback } from 'react'
-import { Redirect, withRouter } from 'react-router-dom'
+
+import * as React from 'react';
```

Comment:
Please check that this respects the current lint/prettier configurations, `;` should be added to all lines or removed from the new ones.